prelude-shim/rules.bzl

```diff
@@ -0,0 +1,4 @@
+genrule = native.genrule
```

Comment:
I think this can be fixed by using `native.genrule` in the right location, as that will work in Buck1 and Buck2. Then you can delete prelude-shim